desc/objectives/_geometry.py

```diff
@@ -451,7 +451,7 @@ class PlasmaVesselDistance(_Objective):
         is fixed or not. If True, the surface is fixed and its coordinates are
         precomputed, which saves on computation time during optimization.
         If False, the surface coordinates are computed at every iteration.
-        False by default.
+        False by default, so that self.things = [eq] only, not [eq,surface]
```

Comment:
I think this is the opposite?

Reply:
Oh yes, this needs to be changed. It should say something like: 

> If True, the surface is fixed and its coordinates are precomputed, which saves on computation time during optimization. In this case `self.things = [eq]` only. If False (default), the surface coordinates are computed at every iteration and `self.things = [eq, surface]`.